spec/services/siade_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Siade, type: :service do
```

Comment:
Clairement pour moi VCR ici c'est overkill, on peut utiliser webmock et flip le jeton de tests, les cassettes and stuff.

Après le souci que j'ai c'est le jeton de test qui pour moi ne sert pas, faudrait soit harden/fine tuner la config VCR, soit carrément flip et utiliser webmock (et ainsi avoir un control fin sur ce qu'on renvoi).

Reply:
Au passage ici ce client devrait gérer de manière générique et globale les erreurs.